x-pack/plugins/ml/public/embeddables/anomaly_charts/anomaly_charts_initializer.tsx

```diff
@@ -23,7 +23,7 @@ import { FormattedMessage } from '@kbn/i18n-react';
 import { AnomalyChartsEmbeddableInput } from '..';
 import { DEFAULT_MAX_SERIES_TO_PLOT } from '../../application/services/anomaly_explorer_charts_service';
 
-export const MAX_ANOMALY_CHARTS_ALLOWED = 48;
+export const MAX_ANOMALY_CHARTS_ALLOWED = 50;
```

Comment:
Not related to your changes here, but in the popover for adding the embeddable if you try to select `0` charts, no warning is displayed next to the input. That only appears if you try to go over 50. Is it possible to prevent selection of 0, or show a warning?

<img width="641" alt="image" src="https://github.com/elastic/kibana/assets/7405507/a3a3232a-357a-4cba-af19-b4abfe1ff178">

Reply:
Fixed here [`db94e50` (#159816)](https://github.com/elastic/kibana/pull/159816/commits/db94e50274d00e38a932148c338aa5aca6d09e17)